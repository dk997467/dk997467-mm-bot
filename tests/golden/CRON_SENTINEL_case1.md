CRON SENTINEL

| check | status | detail |
|-------|--------|--------|
| [x] report_soak_today | OK | verdict=OK |
| [x] full_stack_validation | OK | RESULT=OK seen in last 24h |
| [x] daily_digest | OK | present in last 24h |
| [x] audit_chain | OK | broken=0 in last 24h |
| [x] readiness_score_recency | OK | mtime<7d |
| [x] pre_live_pack_recency | OK | fresh<7d|skip |

SENTINEL=OK


