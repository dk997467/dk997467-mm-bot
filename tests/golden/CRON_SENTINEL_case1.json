{"checks":[{"detail":"verdict=OK","name":"report_soak_today","ok":true},{"detail":"RESULT=OK seen in last 24h","name":"full_stack_validation","ok":true},{"detail":"present in last 24h","name":"daily_digest","ok":true},{"detail":"broken=0 in last 24h","name":"audit_chain","ok":true},{"detail":"skip","name":"readiness_score_recency","ok":true},{"detail":"skip","name":"pre_live_pack_recency","ok":true}],"result":"OK","utc_now":"2024-01-01T23:59:59Z","window_hours":24}
